libs/providers/launchdarkly-client/package.json

```diff
@@ -7,7 +7,7 @@
     "current-version": "echo $npm_package_version"
   },
   "peerDependencies": {
-    "@openfeature/web-sdk": "~0.3.2-experimental",
-    "launchdarkly-js-client-sdk": "^3.1.3"
+    "@openfeature/web-sdk": "*",
```

Comment:
Once we release a non-experimental version, we should properly set this. I'll make an issue to do this for all components in the contribs